.github/workflows/release.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         name: python-package-distributions
         path: dist/
     - name: Publish distribution 📦 to PyPI
-      uses: pypa/gh-action-pypi-publish@release/v1
+      uses: pypa/gh-action-pypi-publish@release/v1.10.0
```

Comment:
I don't know why that one CI step is failing but this is the line it's failing on. I double checked and that branch definitely exists. I wondered if we didn't need the `@release` part, since the tag is available at `v1.10.0`, but I don't know why it would work in the other steps? TL;DR: I looked into why that one CI step failed and whether that mattered or was a red flag, and I'm not sure.

Reply:
Following the documentation they have - it should be `release/v1`, however, considering `release/v*` are branch names, it should be `release/v1.10`